Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,17 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import HeaderContent from "./HeaderContent";
+import "./Header.css";
+
+const HeaderTitle = () => {
+  return (
+    <div className="header-title-wrapper">
+      <div className="header-title">
+        Lambda School
+        <span className="date"> july 31</span>
+        <HeaderContent />
```

Comment:
"HeaderContainer should hold HeaderTitle and HeaderContent. Instead, you put HeaderContent inside of HeaderTitle, which is a little weird. It still works, but the code structure doesn't match what's implied by the names and what's displayed. I'll reference this in the other location as well."

--- See here? HeaderContent should be a sibling, not a child, of HeaderTitle. Everything would still work as you have it if you just imported and used it in the other file. It's a quick fix. ---